src/main/java/games/strategy/engine/framework/ArgParser.java

```diff
@@ -30,14 +29,15 @@ public static boolean handleCommandLineArgs(
         return false;
       }
     }
+    ClientSetting.flush();
```

Comment:
Needed because we call `setSystemProperty` previously. The `flush` is a slow call so it is outside of the for loop.